internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/TestActiveActiveIngestion.java

```diff
@@ -628,6 +644,7 @@ public void testAAIngestionWithStoreView() throws Exception {
             .setMockTime(testMockTime)
             .setRegionName("local-pubsub")
             .build());
+    additionalCloseablesPerMethod.add(localKafka);
```

Comment:
Can we close it at the end of the test on our own? Or using try-with-resource would be better. Using a class variable will mean we need to run this single threaded - or we could run into other race conditions.

Reply:
I think this is not a problem because we do not run tests from the single class concurrently  and hence each test run will have its own `additionalCloseablesPerMethod` (`@BeforeMethod).

Reply:
But why do we want to leave this up to the test framework? Using try-with-resource seems like a much cleaner option to me. I also don't see the benefit of this just yet. Maybe if we were creating resources in different places than where we use them, this kind of a pattern would be helpful.

Reply:
I've really silly argument to defend this. My hestistation to use try-wtih-resource in this case is because it will put rest of the test in the try block and somewhat affects readability.

Reply:
Ugh, yeah. These are culture clashes 😂. Personally, I've gotten used to the try-with-resource and actually, like it quite a bit because you don't need to worry about closing resources when you no longer need them. So, it kind of also nudges us to keep these blocks smaller.

But if you have strong concerns, I can drop this.

Reply:
Well I gave in already! 😄

Reply:
😂 Thanks